src/main/groovy/com/mini/asaas/customer/CompanyType.groovy

```diff
@@ -0,0 +1,48 @@
+package com.mini.asaas.customer
+
+import com.mini.asaas.utils.MessageSourceUtils
+
+enum CompanyType {
+    ASSOCIATION,
+    INDIVIDUAL,
+    LIMITED,
+    MEI,
+    NON_PROFIT_ASSOCIATION
+
+    public Boolean isAssociation() {
+        return this == ASSOCIATION
+    }
+
+    public Boolean isIndividual() {
+        return this == INDIVIDUAL
+    }
+
+    public Boolean isLimited() {
+        return this == LIMITED
+    }
+
+    public Boolean isMei() {
+        return this == MEI
+    }
+
+    public Boolean isNonProfitAssociation() {
+        return this == NON_PROFIT_ASSOCIATION
+    }
+
+    public String getLabel() {
+        return MessageSourceUtils.getEnumLabel(this)
+    }
+
+    public static List<CompanyType> getAvailableForCustomerToChoose() {
```

Comment:
Esse método aqui também, poderia ser implementado a partir do momento que for usado.